packages/playground/src/constants/deployment_list.ts

```diff
@@ -124,6 +124,12 @@ export const deploymentListEnvironments = {
     CHAIN_ENDPOINT: "Chain Endpoint",
   },
 
+  static_website: {
+    SSH_KEY: _ssh,
+    GITHUB_URL: "Clonned Repository URL",
+    ROOT: "Root of clonned Repository ",
```

Comment:
```suggestion
    GITHUB_URL: "Cloned Repository URL",
    ROOT: "Root of cloned Repository ",
```

Reply:
Some of the variables are not being renamed like STATICWEBSITE_DOMAIN. Also, what do you mean by 'Root of cloned Repository'?

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/8964f63a-dcf7-4f30-aa43-0b0950d64a20)